app/controllers/application_controller.rb

```diff
@@ -109,53 +128,51 @@ def store_profile(profile = {})
     # return if current_user.echo_id.nil?
   end
 
-  def store_oauth_token(json = {})
+  def store_oauth_token(oauth_response)
     # URS login
-    session[:access_token] = json['access_token']
-    session[:refresh_token] = json['refresh_token']
-    session[:expires_in] = json['expires_in']
-    session[:logged_in_at] = json.empty? ? nil : Time.now.to_i
-    session[:endpoint] = json['endpoint']
-  end
-
-  def store_session_data(json = {})
-    # Launchpad login
-    session[:launchpad_cookie] = json['launchpad_cookie']
-    session[:auid] = json['auid']
-    session[:launchpad_email] = json['launchpad_email']
-    session[:logged_in_at] = json.empty? ? nil : Time.now.to_i
-    session[:expires_in] = json.fetch('expires_in', 900) # 15 min - Launchpad default time
+    session[:access_token] = oauth_response['access_token']
+    session[:endpoint] = oauth_response['endpoint']
+    session[:expires_in] = oauth_response['expires_in']
+    session[:logged_in_at] = Time.now.to_i
+    session[:refresh_token] = oauth_response['refresh_token']
+    # URS_SESSION_KEYS.each do |session_key|
```

Comment:
why are we leaving this (maybe don't tell me, maybe add a comment)

Reply:
I had wanted some feedback on which to use. I'm going to remove it.